### PR TITLE
Feature/add hiring stories

### PR DESCRIPTION
**Pull request by shellandbull**
This PR adds my hiring story :dart:

### CODE REVIEW

**Comment by binaryberry** [comment]
Cool stuff Mario! :rainbow: :tada: 
Not sure whether that repo is public but I reckon those stories probably should be so people beyond alumni can benefit? Thoughts @nikeshashar ?

Oh and @mariogintili  a small typo l.73 : "the interview will consist off" - it's "of".

**Comment by nikeshashar** [comment]
The repo is open source

File: hiring-stories/mario-gintili.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+Hello guys,
+
+I'm part of February's 2014 cohort. Before that I was an economics student on a gap year in the UK,
+around a month after to work for Workshare as a junior ruby developer at £25,000 PA.
+
+## Before MA
+
+I was studying economics in Caracas, Venezuela. Programming was something I was always interested but never quite
+had the time to master. Since the country was on a bad state(politically and economically) I decided to start a gap
```
Comment (nikeshashar) [inline hiring-stories/mario-gintili.md]: space before bracket

File: hiring-stories/mario-gintili.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+year in the UK and explore the possibility of studying the rest of my major there.
+
+I assessed going for an undergard, but that implied waiting 3 years(in order to be eligible for finance)
```
Comment (nikeshashar) [inline hiring-stories/mario-gintili.md]: space before bracket

File: hiring-stories/mario-gintili.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+and then investing another 3 in a career. Which IMO was wayyyy too much waiting :goberserk: , so I took the wild bet and decided to for MA
+since the career prospectus after completion seemed just as good.
+
+## Getting hired
+
+I'm unfamiliar with your circumstances, but when I was looking for work I needed one ASAP(I was running out of :money_with_wings:)
```
Comment (nikeshashar) [inline hiring-stories/mario-gintili.md]: space before bracket

File: hiring-stories/mario-gintili.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+so I applied for everything that MA pitched at me, after one successful interview I accepted an offer from a company called [Workshare](http://www.workshare.com/)
+
+Around ~3 months after I accepted another offer for a company called [Show My Homework](https://www.showmyhomework.co.uk/) at £32,000 PA since it was:
+
+- A) Less specialised, Workshare was mostly working with Microservices written in Ruby and SMHW , Full-stack Ruby on Rails. IMO Staying in a more general ground the ~1st year of your career is a good thing, keeps thing broad.
+
+- B) More inline with my expectations in terms of salary.
+
+After a year with SMHW, I decided to look for a new job since I wasn't really happy there(new office was far from central london, etc.) and decided to job hunt again, I just now(nov 2015) joined [Zapnito](https://www.zapnito.com/).
```
Comment (nikeshashar) [inline hiring-stories/mario-gintili.md]: space before brackets, (Nov 2015)

File: hiring-stories/mario-gintili.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+When I did the course this department wasn't as developed as I think is today, so I'm guessing is improved quite a lot.
+
+**The good thing** is that you're pushed forward to the technical interview step, so due dilligence is minimal for you.
+
+**But remember** MA's hiring service is like every other recruitment service(but better in some aspects). Which means they only work with their client companies, limiting yourself to them also limits your potential opportunities.
+
+#### Technical interviews.
+
+It's always good to keep yourself sharp by doing [coding challenges](). Knowledge is power here :star:
```
Comment (nikeshashar) [inline hiring-stories/mario-gintili.md]: Is there meant to be a link here?

File: hiring-stories/mario-gintili.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+- Ask your employeer what the interview will consist off, and study/explore more on that topic.
```
Comment (nikeshashar) [inline hiring-stories/mario-gintili.md]: consist of

File: hiring-stories/mario-gintili.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+
+- They will want to see more problem-solving skills and fast learning rather than code. Take your time and feel free to describe how an implementation would look like and work your way to getting it done.
+
+- They are not rocket science, **confidence plays a big role**, I usually come up with better solutions when I'm focused but not intimidated by the problem at hand.
+
+#### Wrapping up.
+
+I'm happy with this career choice, wether I'll come back to finish an undergrad is something that I ask myself every now and then :)
```
Comment (nikeshashar) [inline hiring-stories/mario-gintili.md]: whether

File: hiring-stories/mario-gintili.md (hunk 6 above)
Comment (shellandbull) [inline hiring-stories/mario-gintili.md]: and employer*

**Comment by shellandbull** [comment]
added feedback @nikeshashar . Thanks @binaryberry :smiley:

**Pull request merged**